<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>640</width>
     <height>420</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>420</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>420</height>
    </size>
   </property>
   <widget class="QWidget" name="SynthTab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QScrollArea" name="SynthArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>638</width>
       <height>380</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>638</width>
       <height>380</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>638</width>
       <height>380</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>636</width>
        <height>378</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="XYControl">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>60</height>
    </size>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Vol</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Vol</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="freqLab">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QDial" name="InputVol">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="OutputVol">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
